### PR TITLE
Support CentOS / Redhat + FIX tests

### PR DESCRIPTION
**Pull request by gbloquel**
* FIX unit tests and acceptance test
* Add supported OS in metadata file
* Add travis configuration for Continuous Integration
* Support Centos / Redhat

### CODE REVIEW

File: manifests/service.pp [hunk 1]
```diff
@@ -42,7 +42,8 @@
   $nexus_script = "${nexus_home}/bin/nexus"
 
   if ($::operatingsystem == 'Debian' and versioncmp($::operatingsystemrelease, '8.0') > 0) or
-     ($::operatingsystem == 'Ubuntu' and versioncmp($::operatingsystemrelease, '15.04') > 0) {
+  ($::operatingsystem == 'Ubuntu' and versioncmp($::operatingsystemrelease, '15.04') > 0) or
+  (($::operatingsystem == 'CentOS' or $::operatingsystem == 'RedHat')and versioncmp($::operatingsystemmajrelease, '7') >= 0) {
```
Comment (kenbreeman) [inline manifests/service.pp]: spacing between `)` and `and`

Comment (gbloquel) [inline manifests/service.pp]: Added in the next commit.
Strange because linter and unit test was ok.

**Comment by pranav** [comment]
lgtm

**Comment by gbloquel** [comment]
What is "lgtm"?

**Comment by pranav** [comment]
looks good to me

**Comment by kenbreeman** [comment]
:+1:

**Comment by kenbreeman** [comment]
Released on Forge as 1.7.0, thanks!

https://forge.puppet.com/hubspot/nexus/readme

**Pull request merged**